app/components/offering-editor-learnergroups.js

```diff
@@ -1,21 +1,22 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
 import { isPresent, isEmpty } from '@ember/utils';
-const { sort } = computed;
+import RSVP from 'rsvp';
+
+const { map } = RSVP;
 
 export default Component.extend({
   classNames: ['offering-editor-learnergroups'],
 
   revisedLearnerGroups: computed('cohort.filteredAvailableLearnerGroups.[]', {
```

Comment:
While you are in here anyway can you get rid of the `get()` and `readonly` stuff and just make this looks like all the other CPs?

Reply:
on it.